docs/paper/admin/reference/configuration/world-configuration.mdx

```diff
@@ -0,0 +1,720 @@
+---
+slug: /reference/world-configuration
+---
+
+# World Configuration
+
+import React from 'react';
+import Config from '@site/src/components/Config';
+
+:::info
+
+The below YAML shows you the structure and default values for the world configuration.
+
+Click on a leaf node to view the description for that setting.
+
+:::
+
+<Config
+    data={`
+anticheat:
+  anti-xray:
+    enabled:
+      default: 'false'
+      description: 'Controls the on/off state for the Anti-Xray system. '
+    engine-mode:
+      default: '1'
+      description: 'Sets the Anti-Xray engine mode. 1 replaces specified blocks (hidden-blocks)
+        with other "fake" blocks, stone (deepslate at y < 0), netherrack, or end_stone
+        based on the dimension. In contrast, 2 will replace both hidden-blocks and
+        replacement-blocks with randomly generated hidden-blocks. '
+    max-block-height:
+      default: '64'
+      description: 'Sets the maximum height (y coordinate, starting from the bottom
+        of the world) to which anti-xray will operate. Only integer multiples of 16
+        are accepted. All other values will be rounded down. The [Minecraft Wiki page
+        about Ore](https://minecraft.fandom.com/wiki/Ore) may be helpful in determining
+        the best value for you. '
+    update-radius:
+      default: '2'
+      description: 'Radius for block updates which will be sent containing real block
+        data when the client interacts with a block. Only values 0, 1, and 2 are accepted.
+        Values smaller than 0 will be rounded up to 0, while values larger than 2
+        will be rounded down to 2. 0 is only designed for testing purposes. Do not
+        use it in production. '
+    lava-obscures:
+      default: 'false'
+      description: 'Whether to obfuscate blocks touching lava. Does not work well
+        with non-stone-like ore textures. This is because lava, while being mostly
+        opaque, does not cover blocks fully at the surface. '
+    use-permission:
+      default: 'false'
+      description: 'Whether to allow players with the paper.antixray.bypass permission
+        to bypass anti-xray. Checking this permission is disabled by default as legacy
+        permission plugins may struggle with the number of checks made. This should
+        only be used with modern permission plugins. '
+    hidden-blocks:
+      default: '[copper_ore, deepslate_copper_ore, gold_ore, deepslate_gold_ore, iron_ore,
+        deepslate_iron_ore, coal_ore, deepslate_coal_ore, lapis_ore, deepslate_lapis_ore,
+        mossy_cobblestone, obsidian, chest, diamond_ore, deepslate_diamond_ore, redstone_ore,
+        deepslate_redstone_ore, clay, emerald_ore, deepslate_emerald_ore, ender_chest]'
+      description: 'With engine-mode: 1, these blocks will be replaced by stone (deepslate
+        at y < 0), netherrack, or end_stone, based on the dimension. All types of
+        air are ignored on this list.  With engine-mode: 2, these blocks will be randomly
```

Comment:
There's two space here. In the old config it was a newline. Same happens in the replacement-blocks option